### ISSUE TITLE
Location select box is not accessible

### ISSUE THREAD

**Issue by hursey013** [18F/samhsa-prototype#234]
It's close, it's keyboard accessible, but it is not a good experience for voice software since it does not announce any of the location suggestions. We will need to look into improving the accessibility of the current component or perhaps exploring alternative options.

This library mentioned its WAI-ARIA compliant:https://github.com/hibiken/react-places-autocomplete
